docs/kubernetes_deploy.md

```diff
@@ -79,7 +79,7 @@ images:
 
   triton:
     image: nvcr.io/nvidia/tritonserver
-    tag: 21.12-py3
+    tag: 22,01-py3
```

Comment:
typo?

Reply:
yep

Reply:
I think this was just a test to see if I actually review the changes ;)
Approved